README.md

```diff
@@ -100,14 +101,19 @@ This is the main directive of the module. It's creating a popover with desired a
          icon: "filled-error",
          state: "my-manager.state1"
      }];
+
+     $scope.onSelectOption = function (id) {
+        console.log("Option has been selected!", id);
```

Comment:
`console.log` are not very end-user friendly, maybe adding an error message to scope could be better
```
# in js
$scope.errorMessage =`Option has been selected! ${id}`;
# in html
<div class="red" data-ng-bind="errorMessage"></dev>
```

Reply:
This is only a documentation. As a developer you understand that you can update the $scope if you want, don't you?
This is the same for the other comments.